conductor/src/main/java/com/bluelinelabs/conductor/DefaultControllerFactory.java

```diff
@@ -0,0 +1,16 @@
+package com.bluelinelabs.conductor;
+
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+final class DefaultControllerFactory implements ControllerFactory {
```

Comment:
I think DefaultControllerFactory should implement default logic for creating Controller, but not just be a stub in logic

Reply:
We could only do that without extra reflection if the first argument for `create` was a `Class`.